### PR TITLE
[documented-lasagna]: Updated instructions.

### LINKED ISSUE #578: documented-lasagna: split single task into three tasks
There is currently just one task: `## 1. Document the three methods`

It would be good to split this up into three separate tasks (they're also separate in the tests).

### PR DESCRIPTION
**Pull request by meatball133**
resolves #578 

- Split instructions from 1 task into 3
- Added a little bit more lore
- Put all hints in general

### CODE REVIEW

**Comment by meatball133** [comment]
Hi

> You should describe the content of the function and relevant information, this should be written as a docstring.

Not sure what you mean here, the hints file is the same as before but just put it as a general hint instead of a task-specific.

File: exercises/concept.wip/documented-lasagna/.docs/instructions.md [hunk 1]
```diff
@@ -1,12 +1,22 @@
 # Instructions
 
 In this exercise you're going to document the code you just wrote to help you cook a brilliant lasagna.
+You want to be able to go back to your code and remember what each function does and how to use it.
+You should describe the content of the function and relevant information, this should be written as a docstring.
```
Comment (cmcaine) [inline exercises/concept.wip/documented-lasagna/.docs/instructions.md:6]:
```suggestion
You are unlikely to look at this lasagna-related code ever again, but documenting code is really helpful if you want any larger project to be understandable to others (or to yourself)!

To document your code, write a "docstring" for each of the three functions. Here's how to do it:

- Review the [Documentation][doc] section in the Julia Manual.
- Start with the function signature indented with 4 spaces.
- Include a single one-line sentence describing what the function does, in your own words.
```

File: exercises/concept.wip/documented-lasagna/.docs/instructions.md [hunk 2]
```diff
@@ -1,12 +1,22 @@
...
-Add docstrings to the three methods `preptime`, `remaining_time`, and `total_working_time`.
+The first function you wrote was `preptime`, which calculates how long it takes to prepare the lasagna.
+
+## 2. Document remaining_time
+
+The second function you wrote was `remaining_time`, which calculates how long it is until the lasagna is ready.
+
+## 3. Document total_working_time
+
+The third function you wrote was `total_working_time`, which calculates how long it takes to make the lasagna.
```
Comment (cmcaine) [inline exercises/concept.wip/documented-lasagna/.docs/instructions.md:22]:
```suggestion
The third function you wrote was `total_working_time`, which calculates how long it takes to make the lasagna.

[doc]: https://docs.julialang.org/en/v1/manual/documentation/
```

File: exercises/concept.wip/documented-lasagna/.docs/hints.md [hunk 3]
```diff
@@ -2,8 +2,6 @@
 
 ## General
 
-## 1. Document the three methods
-
 - Review the [Documentation][doc] section in the Julia Manual.
 - Start with the function signature indented with 4 spaces.
 - Include a single one-line sentence describing what the function does, in your own words.
```
Comment (cmcaine) [inline exercises/concept.wip/documented-lasagna/.docs/hints.md:7]: Delete this whole file?

**Review by cmcaine — COMMENTED** [review]
Here's my pitch for what we should do. What do you think?

File: exercises/concept.wip/documented-lasagna/.docs/hints.md (hunk 3 above)
Comment (meatball133) [reply exercises/concept.wip/documented-lasagna/.docs/hints.md:7]: I think you can have general hints, but if your idea is to put the content in the instructions file is there no purpose of having a hints file I would say,

File: exercises/concept.wip/documented-lasagna/.docs/instructions.md [hunk 4]
```diff
@@ -1,12 +1,28 @@
 # Instructions
 
 In this exercise you're going to document the code you just wrote to help you cook a brilliant lasagna.
+You are unlikely to look at this lasagna-related code ever again, but documenting code is really helpful if you want any larger project to be understandable to others (or to yourself)!
 
+To document your code, write a "docstring" for each of the three functions. Here's how to do it:
+
+- Review the [Documentation][doc] section in the Julia Manual.
+- Start with the function signature indented with 4 spaces.
+- Include a single one-line sentence describing what the function does, in your own words.
```
Comment (cmcaine) [inline exercises/concept.wip/documented-lasagna/.docs/instructions.md:10]:
```suggestion
- Include a single one-line sentence describing what the function does, in your own words.

```

**Review by cmcaine — APPROVED** [review]
Seems okay to me, now, thank you! I'll leave it a few days so Sascha can comment if they want.

**Pull request merged**